src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/UsageReport/WriteUsageBurndownData.cs

```diff
@@ -43,30 +44,41 @@ public class WriteUsageBurndownData : Task
         [Required]
         public string OutputFilePath { get; set; }
 
-        public override bool Execute()
+        /// <summary>
+        ///  Sends HTTP requests and receives HTTP responses.
+        /// </summary>
+        private readonly HttpClient client = new HttpClient();
```

Comment:
Since the type is declared on the left side, it doesn't need to be declared again with the `new` keyword.

```suggestion
        private readonly HttpClient client = new();
```

Reply:
Addressed in commit: [Remove type from RHS of declaration](https://github.com/dotnet/installer/pull/17128/commits/3c0ca3d82d48f902237ae78916f9032d09c2737b)